app/helpers/application_helper.rb

```diff
@@ -166,7 +166,8 @@ def responsibles_selector_input(form)
     users = User.active.pluck(:id, :name, :username, :email).map { |user| [user[0], user[1].blank? ? user[2] : user[1], user[2], user[3]] }.sort_by { |user| user[1].downcase }
     form.input :responsibles, as: :select,
                               collection: options_for_select(users.map {|user| ["#{user[1]} (#{user[2]}) #{user[3]}", user[0]]}, @booth.responsibles.map(&:id)),
-                              include_blank: false, label: 'Responsibles', input_html: { class: 'select-help-toggle', multiple: 'true' }
+                              include_blank: false, label: 'Responsibles', input_html: { class: 'select-help-toggle', multiple: 'true' },
+                              hint: 'This field lets you search for users and add them as a responsible for the booth you create. It searches from users that already exist, i.e have an account in osem.'
```

Comment:
Feel free to not copy paste my explanation from the issue and make this sentence shorter ;)